Objects/regions.c

```diff
@@ -959,3 +961,57 @@ PyTypeObject PyRegion_Type = {
     0,                                       /* tp_alloc */
     PyType_GenericNew,                       /* tp_new */
 };
+
+static const char *get_region_name(PyObject* obj) {
+    if (IS_DEFAULT_REGION(obj->ob_region)) {
+        return "Default";
+    } else if (IS_IMMUTABLE_REGION(obj->ob_region)) {
+        return "Immutable";
+    } else {
+        const regionmetadata *md = (regionmetadata *)obj->ob_region;
```

Comment:
Once #11 is merged you can use:

```suggestion
        const regionmetadata *md = Py_REGION_DATA(obj);
```

For now it would be better to use `Py_REGION(obj)` to access the region of an object. Using the access macros will make potential refactorings (Like changing the type of the object field) easier :)

Reply:
Small correction, I thought I've added the macro in that PR but apparently not.

This is the macro I plan to add:

```
#define Py_REGION_DATA(ob) (_Py_CAST(regionmetadata*, Py_REGION(ob)))
```